src/Orleans/Configuration/GlobalConfiguration.cs

```diff
@@ -193,6 +193,10 @@ public bool PrimaryNodeIsRequired
         /// Whether to use the gossip optimization to speed up spreading liveness information.
         /// </summary>
         public bool UseLivenessGossip { get; set; }
+        /// <summary>
+        /// Whether new silo that joins the cluster has to validate the initial connectivity with all other Active silos.
```

Comment:
It might be nice to have a hint on when or why would you want to set this to false